apo.obo

```diff
@@ -141,6 +141,7 @@ subset: CGD
 subset: CryptoGD
 subset: SGD
 synonym: "loss of function" RELATED [SGD:RSN]
+synonym: "null" EXACT []
```

Comment:
Should this maybe be `BROAD`?

Reply:
@suzialeksander if you agree this could be BROAD (or RELATED?) let's do another little PR.